bindings/mnt/pyfiction/include/pyfiction/algorithms/simulation/sidb/random_sidb_layout_generator.hpp

```diff
@@ -83,8 +84,11 @@ inline void random_sidb_layout_generator(pybind11::module& m)
                        DOC(fiction_generate_random_sidb_layout_params_maximal_attempts_for_multiple_layouts));
 
     // NOTE be careful with the order of the following calls! Python will resolve the first matching overload!
+    detail::random_layout_generator<py_charge_distribution_surface_100>(m);
+    detail::random_layout_generator<py_charge_distribution_surface_111>(m);
```

Comment:
Why is it important that the layout could have charge information? Ist that information respected in any way?

Reply:
not really!